app/src/main/java/com/infomaniak/mail/data/models/ai/AiMessage.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Infomaniak Mail - Android
+ * Copyright (C) 2023 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.mail.data.models.ai
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+open class AiMessage(var content: String, val type: String)
```

Comment:
That's what I wanted to do at first but I ended up needing to instantiate AiMessages because of the old api calls for shortcuts. Since we have our own mobile api routes I don't need to instantiate it anymore and I could pass it back as abstract!